public/wp-content/themes/ColoredCow/Register-guest.php

```diff
@@ -0,0 +1,43 @@
+<?php 
+ /**
+ * Template Name: Register_form
```

Comment:
Since it's a name that will be accessible from Dashboard, can we switch to a better naming convention that is friendly to a non-techie? Like "Register Form"?